### PR TITLE
Update to latest pocket-core and setup refactoring

### PR DESCRIPTION
**Pull request by aqt01**

### CODE REVIEW

**Review by julio-jimenez — DISMISSED** [review]
Lgtm

**Review by emedrano2112 — APPROVED** [review]
This changes are looking good to me, but please, wait for @julio-jimenez 's approval

**Review by emedrano2112 — APPROVED** [review]

**Review by emedrano2112 — DISMISSED** [review]

**Review by julio-jimenez — APPROVED** [review]

**Review by oten91 — APPROVED** [review]

**Pull request merged**
